project_event/views/project_task_view.xml

```diff
@@ -55,22 +67,136 @@
         </field>
     </record>
 
+    <record id="project_event_task_form" model="ir.ui.view">
+        <field name="name">Task Form</field>
+        <field name="model">project.task</field>
+        <field name="arch" type="xml">
+            <form string="Task">
+                <header>
+                    <field name="stage_id" widget="statusbar" clickable="True"
```

Comment:
@rimbendhaou Are you using the stage_id for the activity workflow?

Reply:
@joaoalf it was for demo and for the test